packages/es-components/babel.config.js

```diff
@@ -0,0 +1,12 @@
+const production = process.env.NODE_ENV === 'production' || process.env.BABEL_ENV === 'production';
+
+const presets = [
+  ['wtw-im', { extractFormatMessage: false, transformFormatMessage: false }]
+];
+
+const plugins = [
+  '@babel/plugin-proposal-export-default-from',
+  ['babel-plugin-styled-components', { displayName: !production, fileName: !production }]
```

Comment:
The `styled-components` plugin should be included in the [wtw preset](https://github.com/WTW-IM/babel-preset-wtw-im#babel-plugin-styled-components). Does the following not work?

```js
const presets = [
  ['wtw-im', {
    extractFormatMessage: false,
    transformFormatMessage: false,
    styledComponents: { displayName: !production, fileName: !production }
  }]
];
```

Is there a particular reason why we renamed `.baberc.js` to `babel.config.js`? Both do the exact same thing.

Reply:
I was having some configuration issues getting tests to run and read something that said Jest didn't see a .babelrc.js (pretty sure that was outdated info since that wasn't the issue) but I renamed it so it matched ui-components config, and it's more consistent with the other config files in our root.

I just tried your suggestion and it does work with the settings in the wtw preset, though for some reason it makes the generated classname even longer. 

Maybe we don't need the generated classnames either, based on your points. If they won't be useful for styling or testing, they're just added bytes.

Reply:
I'm good getting rid of the class names. Consumers of the components can append their own class name as far as I'm concerned. 

As far as testing is concerned, I think there are other methods for selecting elements available that we can recommend.

Reply:
Purged!